website/docs/components/accordion/partials/code/code-snippets/accordion-a11y-name-component.gts

```diff
@@ -0,0 +1,22 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+
+import { HdsAccordion } from '@hashicorp/design-system-components/components';
+import type { HdsAccordionSignature } from '@hashicorp/design-system-components/components/hds/accordion/index';
+
+interface AccordionA11yNameSignature {
+  Element: HdsAccordionSignature['Element'];
+}
+
+const AccordionA11yName: TemplateOnlyComponent<AccordionA11yNameSignature> =
+  <template>
+    <HdsAccordion ...attributes as |A|>
```

Comment:
why do we need `...attributes` in the `.gts` examples? also, this may cause confusion for the consumers (_"why `hbs` and `gts` are different?"_), so probably we should try to keep them as aligned as possible (also for our own future sanity: we will never remember if some differences are intentional/needed or not)